### ISSUE TITLE
Windows Environment, Can not use Service name to access.

### ISSUE THREAD

**Issue by kingsd041** [rancher/rancher#10578]
Rancher versions:
rancher/server: 1.6.13-rc3
rancher/agent:

Docker version: (docker version,docker info preferred)
17.06.2-ee-6
Operating system and kernel: (cat /etc/os-release, uname -r preferred)
Windows Server 2016 Standard
Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)
VirtualBox
Setup details: (single node rancher vs. HA rancher, internal DB vs. external DB)
single node rancher
Environment Template: (Cattle/Kubernetes/Swarm/Mesos)
**Windows****

**Steps to Reproduce:**
1. Create service "foo1" in stack "bar" 
2. Create service "foo2" in stack "bar" 
3. Login service "foo1" execute shell: 
     `ping foo2`
**Results:**
Can not use Service name to access.

```
PS C:\Program Files> ping foo2
Ping request could not find host foo2. Please check the name and try again.
```

**Comment by deniseschannon** [comment]
Available with v1.6.13-rc4

**Comment by kingsd041** [comment]
Tested with Rancher/server:v1.6.13-rc4
Test passed.
